packages/featured-slider/package.json

```
@@ -3,12 +3,15 @@
   "description": "featured slider for case studies",
   "version": "4.1.5",
   "author": "HashiCorp",
-  "contributors": ["Nicole Forrester"],
+  "contributors": [
+    "Nicole Forrester"
```

I think it makes sense to add this since you have basically rewritten this component. This field I think is mostly helpful in figuring out who to reach out to with questions about components. Especially relevant here given that Nicole doesn't work here anymore 

```suggestion
    "Zach Shilton"
```